tests/integration/commands/status/__snapshots__/status.test.ts.snap

```diff
@@ -13,5 +13,8 @@ exports[`fixture: empty-project > should print status for a linked site 2`] = `
 {
   "Email": "test@netlify.com",
   "Name": "Test User",
+  "Teams": [
+    "Test User",
+  ],
```

Comment:
This accounts API response fixture is now wrong: https://github.com/netlify/cli/blob/f0fdfef9821fbfb806c9648811b5ee9ac0e5d173/tests/integration/commands/status/status.test.ts#L16. Can we update it? Maybe also pull in the `MinimalAccount` type there.